src/lib/style/components/entity_finder/_breadcrumbs.scss

```diff
@@ -10,17 +11,16 @@
   white-space: nowrap;
   overflow-x: auto;
   .BreadcrumbItem {
-    color: SRC.$primary-action-color;
-  }
-  .BreadcrumbItem:hover {
-    text-decoration: underline;
+    @include mixins.normalLink();
+    font-weight: normal;
```

Comment:
Not bold to maintain visual contrast between the "Current" breadcrumb and the rest of the items